packages/react/helpers/hashClassesHelpers.tsx

```
@@ -1,16 +1,18 @@
  * @param styled Apply only if styled props
  * @param classes String classes
  */
 export const hashClass = (styled = false, classes: string): string => {
+  const { hash } = useTrilogyContext()
+
   if (classes.trim().length < 1) return ''
   if (styled) {
     return classes
       .split(' ')
-      .map((classe: string) => (classe = `${classe}__${hash.HASH}`))
+      .map((classe: string) => (classe = `${classe}__${hash}`))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)